torchci/lib/fetchPR.ts

```diff
@@ -20,11 +28,46 @@ export default async function fetchPR(
       pull_number: parseInt(prNumber),
       per_page: 100,
     }),
+    await rocksetClient.queryLambdas.executeQueryLambda(
+      "commons",
+      "pr_commits",
+      rocksetVersions.commons.pr_commits as string,
+      {
+        parameters: [
+          {
+            name: "pr_number",
+            type: "int",
+            value: prNumber,
+          },
+          {
+            name: "owner",
+            type: "string",
+            value: `${owner}`,
+          },
+          {
+            name: "repo",
+            type: "string",
+            value: `${repo}`,
+          },
+        ],
+      }
+    ),
   ]);
   const title = pull.data.title;
-  const shas = commits.map((commit) => {
-    return { sha: commit.sha, title: commit.commit.message.split("\n")[0] };
+
+  console.log("Fetcing PRS ZZZZZZZZZZ")
+  console.log(historicalCommits)
```

Comment:
Is this really a common policy to have such verbose log messages?
```suggestion
```

Reply:
sorry, just realized I didn't push my last commit which cleaned this all up 🫣